input/pagecontent/changes.md

```diff
@@ -1,6 +1,16 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01-24
+- Expected Publication status: Normative
+- Based on FHIR version: 4.0.1
+
+This change log documents the significant updates and resolutions implemented from version 1.0.0-ballot to 1.0.0.
+
+#### Changes in this version
+- Removed Must Support flag and associated Obligation extensions from Location.identifier [FHIR-47105](https://jira.hl7.org/browse/FHIR-47105) and Location.mode [FHIR-47104](https://jira.hl7.org/browse/FHIR-47104)
```

Comment:
Please include the profile name in this change log entry (see what's done in other entries).